### PR TITLE
Merch in show

### PR DESCRIPTION
**Pull request by amsalmeron**
Add a Merchant Invoice Show that displays items per an invoice for a merchant.

### CODE REVIEW

File: app/models/invoice_item.rb [hunk 1]
```diff
@@ -2,5 +2,5 @@ class InvoiceItem < ApplicationRecord
     belongs_to :invoice
     belongs_to :item
 
-    enum status: ['pending', 'packaged', 'shipped']
+    enum status: ['Pending', 'Packaged', 'Shipped']
```
Comment (z-prince) [inline app/models/invoice_item.rb:5]: Love the caps here for sure!

File: app/views/merchant_invoices/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<h1>Merchant Invoice Show Page</h1>
+
+<h3><%= @invoice.invoice_items.each do |invoice_item| %></h3>
+    <div id="invoice-item-<%= invoice_item.item.id %>">
```
Comment (z-prince) [inline app/views/merchant_invoices/show.html.erb:4]: Way to iterate divs here!

File: spec/features/merchants/invoices/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'Merchant Invoice Show Page' do
```
Comment (z-prince) [inline spec/features/merchants/invoices/show_spec.rb:3]: Very beefy and also testing sad paths!

File: spec/features/merchants/items/show_spec.rb [hunk 4]
```diff
@@ -34,15 +25,6 @@
   end
 
   it "shows a link to update item information and displays a flash message when successful" do
-    # As a merchant,
```
Comment (z-prince) [inline spec/features/merchants/items/show_spec.rb:37]: Way to get rid of all the stories!

**Review by z-prince — APPROVED** [review]

**Comment by jimriddle1** [comment]
Nice work!

**Pull request merged**